cli/src/program.rs

```diff
@@ -2275,7 +2319,16 @@ fn do_process_program_write_and_deploy(
         } else {
             loader_instruction::write(buffer_pubkey, loader_id, offset, bytes)
         };
-        Message::new_with_blockhash(&[instruction], Some(&fee_payer_signer.pubkey()), &blockhash)
+        if let Some(compute_unit_price) = compute_unit_price {
+            ixs.push(ComputeBudgetInstruction::set_compute_unit_limit(
+                3 as u32 * 200_000u32,
+            ));
+            ixs.push(ComputeBudgetInstruction::set_compute_unit_price(
+                0u64.mul((10 as u64).pow(6)),
```

Comment:
Will always be 0?

Reply:
```suggestion
                compute_unit_price.mul((10 as u64).pow(6)),
```

Reply:
> Will always be 0?

Oh my bad , that wrong it should be compute_unit_price

Reply:
resolved